letsencrypt/cli.py

```diff
@@ -845,14 +845,17 @@ def _handle_exception(exc_type, exc_value, trace, args):
 
         if issubclass(exc_type, errors.Error):
             sys.exit(exc_value)
-        elif args is None:
-            sys.exit(
-                "An unexpected error occurred. Please see the logfile '{0}' "
-                "for more details.".format(logfile))
         else:
-            sys.exit(
-                "An unexpected error occurred. Please see the logfiles in {0} "
-                "for more details.".format(args.logs_dir))
+            # Tell the user a bit about what happened, without overwhelming
+            # them with a full traceback
+            msg = ("An unexpected error occurred.\n" +
+                   traceback.format_exception_only(exc_type, exc_value)[0] +
```

Comment:
I'm not so sure... this shows very nasty internal errors to the end user, e.g:

```python
>>> traceback.format_exception_only(KeyError, KeyError('asd'))[0]
"KeyError: 'asd'\n"                                          
```

or even more cryptic:

```python
>>> traceback.format_exception_only(OpenSSL.SSL.ZeroReturnError, OpenSSL.SSL.ZeroReturnError())[0]
'ZeroReturnError\n'                                                                           
```

Do we really want to show this? In case we want to debug user's problems, we will have to ask for the full traceback anyway. AFAIR the whole point of @bmw work was to eliminate this absolutely meaningless, internal errors output.

Since, I don't see any benefit for the end-user, maybe you wanted to make a development easier? There is a handy `--debug`, which you can put in `~/.config/letsencrypt/cli.ini` or shell alias to save typing.